frontend/src/views/Home.vue

```diff
@@ -125,13 +126,13 @@ export default {
         if (responseBody.status === 'error') {
           const errorMessage = responseBody.message
           let additionalInfo = ''
-          if (errorMessage.startsWith('Invalid Username')) {
-            additionalInfo =
-              'If you forgot your password please try the following link: <a href="https://healthnetbc.hlth.gov.bc.ca/?resetPassword" target="_blank">https://healthnetbc.hlth.gov.bc.ca/?resetPassword</a><br/>If you are still having trouble, please contact the helpdesk at <b>(250) 952-1234</b> or <b>hlth.helpdesk@gov.bc.ca</b>'
-          } else if (errorMessage.startsWith('User has no role')) {
-            additionalInfo = 'Please contact your access administrator to confirm your access to MSP Direct'
-          } else {
-            additionalInfo = 'If you believe you are seeing this message in error, please contact the helpdesk at: <b>(250) 952-1234</b> or <b>hlth.helpdesk@gov.bc.ca</b>'
+          if (!errorMessage.startsWith('UAT01')) {
+            if (errorMessage.startsWith('Invalid Username')) {
+              additionalInfo =
+                'If you forgot your password please try the following link: <a href="https://healthnetbc.hlth.gov.bc.ca/?resetPassword" target="_blank">https://healthnetbc.hlth.gov.bc.ca/?resetPassword</a><br/>If you are still having trouble, please contact the helpdesk at <b>(250) 952-1234</b> or <b>hlth.helpdesk@gov.bc.ca</b>'
+            } else if (errorMessage.startsWith('User has no role')) {
+              additionalInfo = 'Please contact your access administrator to confirm your access to MSP Direct'
+            } else additionalInfo = 'If you believe you are seeing this message in error, please contact the helpdesk at: <b>(250) 952-1234</b> or <b>hlth.helpdesk@gov.bc.ca</b>'
```

Comment:
1, I think we can remove some of the nesting and simplify things by replacing the outer reference to UAT01 with a final else if (!errorMessage.startsWith('UAT01')) instead of the else block.
2. The final else if block should have braces.